frontend/src/pages/ContributionView.jsx

```
@@ -0,0 +1,286 @@
+            </div>
+
+            {/* Content Tabs */}
+            <div className="content-tabs">
+              <nav className="tabs-nav">
+                {['Transcript', 'Notes', 'Downloads'].map((tab) => (
+                  <button
+                    key={tab}
+                    onClick={() => setActiveTab(tab)}
+                    className={`tab-button ${activeTab === tab ? 'active' : ''}`}
+                  >
+                    {tab}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)